### PR TITLE
[WIP] Revert Floydmats

### LINKED ISSUE #367: Circuit printer saying "You need iron to build that" and not building with iron avaliable.
## Round ID: 
6543
## Reproduction:
You can only build assemblies with a circuit imprinter made from a protolathe. Also every time I first tried to make a "gas vent" in Atmospherics.

### LINKED ISSUE #478: Fix integrated circuits
**BELOW**
## Reproduction: just go thru the printer and try to print stuff. Assemblies seem fine tho

Certain Items in the printer wont print even with a full printer

**ABOVE**

### PR DESCRIPTION
**Pull request by qwertyquerty**
## About The Pull Request

This removes the terrible awful floydmats

## Why It's Good For The Game

They were just a dumb gimmick that used a lot of memory and broke a lot of things.

Fixes #367
Fixes #478

## Changelog
:cl:
del: Removed floydmats
fix: Circuit printers now work
/:cl:

### CODE REVIEW

**Comment by qwertyquerty** [comment]
This one is gonna require a bit of testin

**Comment by qwertyquerty** [comment]
I've reviewed each file manually by hand, and everything looks good but I have done no local tests as of the time of this message, so do not testmerge yet.

**Comment by ike709** [comment]
image
image
no

File: code/datums/components/radioactive.dm [hunk 1]
```diff
@@ -76,8 +76,6 @@
 /datum/component/radioactive/proc/rad_attack(datum/source, atom/movable/target, mob/living/user)
 	radiation_pulse(parent, strength/20)
 	target.rad_act(strength/2)
-	if(!hl3_release_date)
```
Comment (ike709) [inline code/datums/components/radioactive.dm:79]: ???

File: code/game/objects/items.dm [hunk 2]
```diff
@@ -7,181 +7,111 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 /obj/item
 	name = "item"
 	icon = 'icons/obj/items_and_weapons.dmi'
-
-	/// The icon state for the icons that appear in the players hand while holding it. Gotten from /client/var/lefthand_file and /client/var/righthand_file
```
Comment (ike709) [inline code/game/objects/items.dm:11]: Why'd you delete all the comments in this file?

Comment (qwertyquerty) [reply code/game/objects/items.dm:11]: Mistake, will fix later

File: code/datums/components/radioactive.dm (hunk 1 above)
Comment (qwertyquerty) [reply code/datums/components/radioactive.dm:79]: Whoops

**Comment by ike709** [comment]
No testmerge until you undo the rad changes.

**Comment by Guestratem** [comment]
if this fixes circuits I will fly to wherever you live and kiss you

**Comment by Toxici11i** [comment]
floydmats shouldntve existed ever and im not exactly sure why it was merged

**Comment by Toxici11i** [comment]
infact - i dont think think originally floydmats was voted upon being added

**Comment by ike709** [comment]
Conflicts.

**Pull request closed without merging**